cmd/swctl/app/cmd_dependency.go

```diff
@@ -755,33 +755,30 @@ func isUserRoot() (bool, error) {
 	return true, nil
 
 }
-func postInstall(cli Cli) []error {
-	var errorGroup []error
+func dockerPostInstall(cli Cli) error {
+	var err error
 	sudoName, err := logname(cli)
-	fmt.Fprintln(cli.Out(), sudoName)
-	// handling case when linux has no login name (e.g. container)
-	if err == nil {
-		commands := []string{
-			"usermod -aG docker " + sudoName, // add user do docker group
+	logrus.Tracef(sudoName)
```

Comment:
Changing to logrus.Trace is good, the user won't see it by default. However if you run it as developer with trace logging then you get a bunch of output where you guess what output relates to what code line. This can be fine for fast fake debugging (using log output instead of stopping the code flow with debugger and examining variable state with debugger). However when the count of such logging messages grow, it easy becomes hard to track the code line for trace output. The same counts for errors. Therefore i wrap error extensively and use text prefixes also in trace logging. It don't have to by anything fancy. Simple `logrus.Tracef("sudoname: %s", sudoname)` (i would prefer something more fancy like `logrus.Tracef("detected login name: %s", sudoname)`, but that is just me and the simple one serves the purpose of getting right away to the right code line from log text)

PS: the same with the other trace usage